### DISCUSSION TITLE
Shouldn't you and z-ink join forces?

### DISCUSSION THREAD

**Discussion by AN1MATEK** [Audionut/Upload-Assistant#4] [Ideas]
There is another well mantained fork of L4G's script: https://github.com/z-ink/Uploadrr

@z-ink is also doing a great job over there. I wonder if you guys would are aware of this and should maybe join forces into one fork. 

Pinging you @Audionut bc idk if you see it otherwise.

I hope I am not overreaching? 

Thank you for your development efforts!

**Comment by Audionut** [comment]
Hi,

I'm not interested in working with those who push the work of others as their own, or whom add tracker advertisements to open source code.

**Reply by AN1MATEK** [reply]
I surely respect that. Thanks for the infos.
